astropy/table/column.py

```diff
@@ -504,7 +503,7 @@ def get_sortable_arrays(self):
 
 
 class BaseColumn(_ColumnGetitemShim, np.ndarray):
-    meta = MetaData(default_factory=OrderedDict)
+    meta = MetaData(default_factory=dict)
     # It's important that the metadata is an OrderedDict so that the order of
```

Comment:
Can you adjust this comment? Or perhaps just remove it?

Reply:
Removed.